snippets/README.md

```diff
@@ -0,0 +1,10 @@
+# Teachable Machine export snippets
+
+This folder contains markdown snippets that are being displayed inside the export panel inside [Teachable Machine](https://teachablemachine.withgoogle.com/). The snippets contain code and instructions on how to use the exported models from Teachable Machine in languages like Javascrip, Java and Python. The folder is split up into folders for each type of mode - image, pose and audio, and within those folders split up into different model export types, like tensorflow js and tflite. 
```

Comment:
Javascrip should be Javascript

Reply:
should we say "for each type of modeL"?

Reply:
"This SECTION is split up into folders for each type of model - image, pose and audio, and within those folders split up into different model export types, like tensorflow js and tflite."